### PR TITLE
[RDY] Spellcheck 2

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
Continues #22882.

# What was changed:

- multiple typos were fixed in the comments and strings.

### CODE REVIEW

File: src/character.cpp [hunk 1]
```diff
@@ -1922,7 +1922,7 @@ hp_part Character::body_window( const std::string &menu_header,
     trim_and_print( hp_window, 1, 1, getmaxx(hp_window) - 2, c_light_red, menu_header.c_str() );
     const int y_off = 2; // 1 for border, 1 for header
 
-    /* This struct estabiles some kind of connection between the hp_part (which can be healed and
+    /* This struct estabilishes some kind of connection between the hp_part (which can be healed and
```
Comment (boydkr) [inline src/character.cpp]: `establishes`

File: src/iexamine.cpp [hunk 2]
```diff
@@ -2233,7 +2233,7 @@ void iexamine::keg(player &p, const tripoint &examp)
             add_msg(m_info, _("You don't have any drinks to fill the %s with."), g->m.name(examp).c_str());
             return;
         }
-        // Make lists of unique drinks... about third time we do this, maybe we oughta make a function next time
```
Comment (boydkr) [inline src/iexamine.cpp]: `wanna` is ok, but not `oughta` ;)

File: src/mondeath.cpp [hunk 3]
```diff
@@ -674,7 +674,7 @@ void mdeath::detonate(monster *z)
         pre_dets.push_back(tmp);
     }
 
-    // Update any hardcoded explosion equivalencies
```
Comment (boydkr) [inline src/mondeath.cpp]: This isn't actually a typo

**Comment by boydkr** [comment]
Nice work. how did you even find all these?

**Comment by ZhilkinSerg** [comment]
> Nice work. how did you even find all these?

Visual Studio Community Edition and this plugin - https://github.com/EWSoftware/VSSpellChecker

File: src/iexamine.cpp (hunk 2 above)
Comment (ZhilkinSerg) [reply src/iexamine.cpp]: I don't wanna "oughta", but I ought to replace "want to". ;)

**Pull request merged**